Monocle is not my cup of tea

--- a/dwm.c 2015-08-01 19:49:50.305463478 +0200
+++ b/dwm.c 2015-08-01 19:49:50.305463478 +0200
@@ -203,7 +203,6 @@
 static void manage(Window w, XWindowAttributes *wa);
 static void mappingnotify(XEvent *e);
 static void maprequest(XEvent *e);
-static void monocle(Monitor *m);
 static void motionnotify(XEvent *e);
 static void movemouse(const Arg *arg);
 static Client *nexttiled(Client *c);
@@ -1238,20 +1237,6 @@
 }
 
 void
-monocle(Monitor *m) {
-	unsigned int n = 0;
-	Client *c;
-
-	for(c = m->clients; c; c = c->next)
-		if(ISVISIBLE(c))
-			n++;
-	if(n > 0) /* override layout symbol */
-		snprintf(m->ltsymbol, sizeof m->ltsymbol, "[%d]", n);
-	for(c = nexttiled(m->clients); c; c = nexttiled(c->next))
-		resize(c, m->wx, m->wy, m->ww - 2 * c->bw, m->wh - 2 * c->bw, False);
-}
-
-void
 motionnotify(XEvent *e) {
 	static Monitor *mon = NULL;
 	Monitor *m;
@@ -1400,10 +1385,7 @@
 	XWindowChanges wc;
 
     if(c->isfloating || selmon->lt[selmon->sellt]->arrange == NULL) { globalborder = 0 ; }
-    else {
-        if (selmon->lt[selmon->sellt]->arrange == monocle) { globalborder = 0 - borderpx ; }
-        else { globalborder =  gappx ; }
-    }
+    else globalborder =  gappx ;
 
     c->oldx = c->x; c->x = wc.x = x + globalborder;
     c->oldy = c->y; c->y = wc.y = y + globalborder;
