Draw colorized underline under the active tag

--- a/dwm.c	2015-02-28 19:49:50.305463478 +0200
+++ b/dwm.c	2015-02-28 19:49:50.305463478 +0200
@@ -733,6 +733,8 @@
 			urg |= c->tags;
 	}
 	dc.x = 0;
+    XSetForeground(dpy, dc.gc, dc.colors[0][ColBG]);
+    XFillRectangle(dpy, dc.drawable, dc.gc, dc.x, dc.y, sw, bh);
 	for(i = 0; i < LENGTH(tags); i++) {
 		dc.w = TEXTW(tags[i]);
         col = dc.colors[ (m->tagset[m->seltags] & 1 << i) ?
@@ -740,6 +742,8 @@
         drawtext(tags[i], col, True);
 		/*drawsquare(m == selmon && selmon->sel && selmon->sel->tags & 1 << i,
 		           occ & 1 << i, col);*/
+        XSetForeground(dpy, dc.gc, col[ColBorder]);
+        XFillRectangle(dpy, dc.drawable, dc.gc, dc.x, dc.h - taglinepx, dc.w, taglinepx);
 		dc.x += dc.w;
 	}
 	dc.w = blw = TEXTW(m->ltsymbol);
